plugins/hdfs_assetstore/web_client/templates/hdfs_assetstore_editFields.jade

```diff
@@ -0,0 +1,16 @@
+.form-group
+  label.control-label(for="g-edit-hdfs-host") Namenode host
+  input.input-sm#g-edit-hdfs-host.form-control(type="text" placeholder="Host")
+.form-group
+  label.control-label(for="g-edit-hdfs-port") Namenode RPC port
+  input.input-sm#g-edit-hdfs-port.form-control(type="text"
+      placeholder="RPC port")
+.form-group
+  label.control-label(for="g-edit-webhdfs-port") WebHDFS port (default is 50700)
+  input.input-sm#g-edit-webhdfs-port.form-control(type="text" placeholder="WebHDFS port")
+.form-group
+  label.control-label(for="g-edit-hdfs-path") Root path
+  input.input-sm#g-edit-hdfs-path.form-control(type="text" placeholder="Root path")
+.form-group
+  label.control-label(for="g-edit-hdfs-user") Effective user (default is current user)
+  input.input-sm#g-edit-hdfs-user.form-control(type="text" placeholder="Username")
```

Comment:
I think these fields should be in the same order as the fields in `hdfs_assetstore_create.jade`.  For instance, Root Path is after host here but before it in the create area.